<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonSettings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="icon">
     <iconset resource="../../../assets/resources.qrc">
      <normaloff>:/settings-hot.png</normaloff>:/settings-hot.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonExit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="icon">
     <iconset resource="../../../assets/resources.qrc">
      <normaloff>:/power-hot.png</normaloff>:/power-hot.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonToggleCursor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../../assets/resources.qrc">
      <normaloff>:/cursor-hot.png</normaloff>:/cursor-hot.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSleep">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="icon">
     <iconset resource="../../../assets/resources.qrc">
      <normaloff>:/sleep-hot.png</normaloff>:/sleep-hot.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>pushButtonToggleCursor</tabstop>
  <tabstop>pushButtonSettings</tabstop>
  <tabstop>pushButtonExit</tabstop>
 </tabstops>
 <resources>
  <include location="../../../assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
